store/blocks.json

```diff
@@ -50,6 +50,16 @@
       "login-content#default"
     ]
   },
+  "store.orderplaced": {
+    "blocks": [
+      "order-placed"
+    ]
+  },
+  "header": {
```

Comment:
Do we need that? Maybe we can add

```
parent: {
  header: header.full
}
```

To the store.orderplaced block

Reply:
That does work, but maybe we don't want the full header on the orderPlaced page? There may also be other custom pages that use header instead of header.full, since header is the default "before" block.